target_clickhouse/sinks.py

```diff
@@ -50,6 +51,17 @@ def full_table_name(self) -> str:
             db_name=self.database_name,
         )
 
+    def clickhouse_table_engine(self) -> str | None:
```

Comment:
We're already passing the engine as [config to the target component](https://github.com/PicnicSupermarket/picnic-dbt-models/blob/813e69ccbc604dadfca84f2f3e1701f33d8b6e2a/rti-models/meltano-streams/meltano.yml#L676), no need to fetch it from the database.